cmake/common.cmake

```diff
@@ -45,6 +45,11 @@ function(HIDE_DISTRACTING_VARIABLES)
     mark_as_advanced(CMAKE_OSX_DEPLOYMENT_TARGET)
     mark_as_advanced(CMAKE_OSX_SYSROOT)
   endif()
+  if(WIN32)
+    mark_as_advanced(LibXml2_DIR)
```

Comment:
I think on windows we are going to have to have this variable visible in the standard view.  It is going to be needed when using a CMake GUI configuration tool which is the default way of configuring CMake projects on Windows.

Reply:
I just want things to be clean and that wouldn't be the case here. Also, let's be serious, to use the advanced mode is really not such a big deal. If people can't be bothered to do that, then they should use the binaries that I imagine we will be making available at some point.

Reply:
If windows users always have to use the advanced mode, then it's never clean for them

Reply:
It depends on your definition of clean. For me, it's when I only see things that are directly related to libCellML. So, `LibXml2_DIR` doesn't fall in that category. Also, to have `LibXml2_DIR` in the main mode means that it wouldn't be consistent with what we would be doing on Linux/macOS.

Also, I have just noticed that we actually get a few more variables: `LIBXML2_INCLUDE_DIR`, `LIBXML2_LIBRARY` and `LIBXML2_XMLLINT_EXECUTABLE`.

![Image2](https://user-images.githubusercontent.com/602265/54463896-7355f800-47d9-11e9-9123-2d529229fcb9.png)

So, it actually gets quite busy and somewhat confusing since the `LIBXML2_XXX` variables all have a value of `LIBXML2_XXX-NOTFOUND` even though everything is "properly" configured.